src/test/kotlin/com/igorwojda/integer/pyramidgenerator/challenge.kt

```diff
@@ -4,7 +4,20 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 fun generatePyramid(n: Int): List<String> {
-    TODO("not implemented")
+    var list = mutableListOf<String>()
+    for (i in 0 until n){
```

Comment:
Can you think of a way to construct the list while iterating instead of having a mutable list that you keep on appending?